source/models/bounties/Bounty.js

```diff
@@ -0,0 +1,124 @@
+﻿const { EmbedBuilder, Guild } = require('discord.js');
+const { DataTypes: { BIGINT, STRING, INTEGER, BOOLEAN }, Model } = require('sequelize');
+const { ihpAuthorPayload } = require('../../embedHelpers');
+const { Hunter } = require('../users/Hunter');
+const { Guild: HunterGuild } = require('../guilds/Guild');
+const { database } = require('../../../database');
+
+/** Bounties are user created objectives for other server members to complete */
+const bountyModel = {
+	id: {
+		primaryKey: true,
+		type: BIGINT,
+		autoIncrement: true
+	},
+	userId: {
+		type: STRING,
+		references: {
+			model: 'User',
+			key: 'id'
+		}
+	},
+	guildId: {
+		type: STRING,
+		references: {
+			model: 'Guild',
+			key: 'id'
+		}
+	},
+	postingId: {
+		type: STRING,
+	},
+	slotNumber: {
+		type: INTEGER,
+		allowNull: false
+	},
+	isEvergreen: {
+		type: BOOLEAN,
+		devaultValue: false
+	},
+	title: {
+		type: STRING,
+		allowNull: false,
+		defaultValue: "\u200B"
+	},
+	description: {
+		type: STRING,
+		allowNull: false,
+		defaultValue: "\u200B"
```

Comment:
Do these need to be zero-width spaces?

Reply:
Removed in https://github.com/Imaginary-Horizons-Productions/BountyBot/pull/23/commits/d01fbba08ed572cf8539025c696959361cd08b91